app/assets/javascripts/models/attachment_options.js

```diff
@@ -0,0 +1,63 @@
+export default class AttachmentOptions {
+
+  constructor({ refreshCallback }) {
+    this.refreshTime = 3600000;
+    this.timestampVerifierId;
```

Comment:
You don't need this line, JavaScript doesn't expect you to declare the class/instance attributes.